src/heretic/mn_menu.c

```diff
@@ -2220,7 +2220,8 @@ boolean MN_Responder(event_t * event)
 
     if (!MenuActive)
     {
-        if (key == key_menu_activate || gamestate == GS_DEMOSCREEN || demoplayback)
+        //[crispy] don't pop up the menu on other keys during a demo
```

Comment:
Nit-pick, but please add a space between `//` and the actual comment (on both lines).